src/main/java/frc/robot/Robot.java

```diff
@@ -21,13 +23,14 @@
 public class Robot extends TimedRobot {
     // Singleton Instances
     private AutonSelector autonSelector = AutonSelector.getInstance();
+    private DelaySelector delaySelector = DelaySelector.getInstance();
```

Comment:
Since delay selection is really part of auton selection, it doesn't need its own class, it can be build as a second item inside of auton selector